src/main/java/ch/njol/skript/expressions/ExprEnchantments.java

```diff
@@ -40,24 +31,40 @@
 import ch.njol.skript.util.EnchantmentType;
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.meta.EnchantmentStorageMeta;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 
 @Name("Item Enchantments")
-@Description("All the enchantments an <a href='classes.html#itemtype'>item type</a> has.")
-@Examples("clear enchantments of event-item")
-@Since("2.2-dev36")
+@Description("All the [stored] enchantments an <a href='classes.html#itemtype'>item type</a> has.")
+@Examples({
+	"clear enchantments of event-item",
+	"add unbreaking 2 to enchantments of player's tool # will only remove unbreaking with level 2",
+	"remove sharpness from enchantments of player's tool # doesn't care about enchantment level",
+	"set {_storedEnchantments::*} to stored enchantments of player's tool # for enchanted books"
+})
+@Since("2.2-dev36, INSERT VERSION (stored enchantments)")
 public class ExprEnchantments extends SimpleExpression<EnchantmentType> {
 
 	static {
-		PropertyExpression.register(ExprEnchantments.class, EnchantmentType.class, "enchantments", "itemtypes");
+		PropertyExpression.register(ExprEnchantments.class, EnchantmentType.class, "[:stored] enchantments", "itemtypes");
 	}
 
 	@SuppressWarnings("null")
 	private Expression<ItemType> items;
+	private boolean isStored;
 
-	@SuppressWarnings({"null","unchecked"})
 	@Override
+	@SuppressWarnings({"null","unchecked"})
```

Comment:
```suggestion
	@SuppressWarnings("unchecked")
```
i don't think this is needed anymore